searx/static/themes/simple/gruntfile.js

```diff
@@ -137,6 +127,30 @@ module.exports = function(grunt) {
         }
       },
     },
+    image: {
+      svg4web: {
+        options: {
+          svgo: ['--config', 'svg4web.svgo.js']
+        },
+        files: {
+          '<%= _templates %>/__common__/searxng-wordmark.min.svg': '<%= _brand %>/searxng-wordmark.svg'
+        }
+      }
+    },
+    svg2png: {
+      favicon: {
+        options: {
+          width: 64,
+          hight: 64
+        },
+        files: [
+          {
+            src: '<%= _brand %>/searxng-wordmark.svg',
+            dest: [ 'img/favicon.png' ]
```

Comment:
The pattern
```
        files: {
          'img/favicon.png': '<%= _brand %>/searxng-wordmark.svg'
        }
```
make it easy to read that `'<%= _brand %>/searxng-wordmark.svg'` is used in this task and the one just above.